modules/britepoolIdSystem.js

```diff
@@ -84,7 +85,7 @@ export const britepoolIdSubmodule = {
   },
   /**
    * Helper method to create params for our API call
-   * @param {SubmoduleParams} [submoduleConfigParams]
+   * @param {SubmoduleConfig} [submoduleConfigParams]
```

Comment:
I don't think you should have changed this one?

Reply:
you're right